### DISCUSSION TITLE
June 4 Discussion

### DISCUSSION THREAD

**Discussion by tonyfast** [Quansight/writers-workshop#12] [New class]
# the goal of the workshop

* multi media collages

## teacher's preferred ground rules

* How to use the raise handle feature? Keeping the professor on track..
* I encourage y'all to interrupt me when I'm talking or demo'ing. 
  Sometimes I get distracted by my own special interests and this class if for you all.
* To help take notes, please respond to this discussion item and continue editing your one reply during class. i can see the changes on the discussion board as they happen.

## [call to action] :+1: :-1:  :smile: :confused:  :rocket:  :heart:  :eyes: :tada: attendance

react to this discussion so we can take attendance.

![image](https://user-images.githubusercontent.com/4236275/117461813-fd163180-af1b-11eb-901b-bfdee65d8bdc.png)

### Q1 how's your energy

* :+1: i feel energetic from this week
* :-1: i feel exhausted from this week

### Q2 how do you want to participate

* :heart:   i'm happy to be at the writers workshop this week
* :eyes: i want to just watch and listen
* :tada: i am okay being called on for discussions

## agenda

0. [:00-:07] chit chat
1. discuss the last writing assigment on magics.
2. explore placing different forms of media in the notebook

4. [:50-:60] wrapup

   create a work that combines at least two forms of media besides text

**Comment by isabela-pf** [comment]
Homework:
- Use text and multimedia and tell a story.

**Comment by MarsBarLee** [comment]
you can use [ipython display](https://ipython.readthedocs.io/en/stable/api/generated/IPython.display.html)
- audio
- images
- maps

**Reply by tonyfast** [reply]
* javascript/css/html
* videos
* diagrams
